src/liger_kernel/ops/rms_norm.py

```diff
@@ -195,7 +213,7 @@ def rms_norm_forward(X, W, eps, offset, casting_mode):
 
     shape = X.shape
     dim = shape[-1]
-    X = X.view(-1, dim)
+    X = X.reshape(-1, dim)
```

Comment:
why view -> reshape. same question for other parts

Reply:
All these reshape can swap back to view. The change was initially made because I thought my very first implementation was wrong due to not contiguous, and [reshape](https://pytorch.org/docs/stable/generated/torch.reshape.html#torch.reshape) can guarantee tensor contiguous.

All tensors are already contiguous with decorator `@ensure_contiguous`, so view should be fine.